ui/hooks/bridge/useTokensWithFiltering.ts

```diff
@@ -19,7 +19,7 @@ import {
 import { AssetType } from '../../../shared/constants/transaction';
 import { isNativeAddress } from '../../pages/bridge/utils/quote';
 import { CHAIN_ID_TOKEN_IMAGE_MAP } from '../../../shared/constants/network';
-import { Token } from '../../components/app/assets/token-list/token-list';
+import { Token } from '../../components/app/assets/types';
```

Comment:
@MetaMask/swaps-engineers I've moved these types to the root of the `assets` directory